app/routables/index/template.hbs

```diff
@@ -1,4 +1,38 @@
-<main
+<main id="homepage">
+  <section id="hero">
+    <div id="menu-wrap">
+      <input
+        type="checkbox"
+        class="toggler"
+      >
+      <div class="hamburger">
+        <div></div>
+      </div>
+      <div class="menu">
+        <div class="one">
+          <div class="two">
+            <ul>
+              <li><a href="#">Home</a>
+              </li>
+              <li><a href="#">About</a>
+              </li>
+              <li><a href="#">Shop</a>
+              </li>
+              <li><a href="#">Contact</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </div>
+    <h1>Weekend Warrior - カスタムクラブ</h1>
```

Comment:
I'm liking the katakana's haha ⚔️

Reply:
haha, thank you! Thought it gives the page a uniqueness.